lib/compress/zstd_compress.c

```diff
@@ -275,23 +546,43 @@ size_t ZSTD_estimateCCtxSize(ZSTD_compressionParameters cParams)
     return sizeof(ZSTD_CCtx) + neededSpace;
 }
 
+size_t ZSTD_estimateCStreamSize(ZSTD_compressionParameters cParams)
+{
+    size_t const CCtxSize = ZSTD_estimateCCtxSize(cParams);
+    size_t const blockSize = MIN(ZSTD_BLOCKSIZE_MAX, (size_t)1 << cParams.windowLog);
+    size_t const inBuffSize = ((size_t)1 << cParams.windowLog) + blockSize;
+    size_t const outBuffSize = ZSTD_compressBound(blockSize) + 1;
```

Comment:
No idea.
I just copied it "as is" from current `ZSTD_initCStream()` :
https://github.com/facebook/zstd/blob/dev/lib/compress/zstd_compress.c#L3220

This could be to ensure some kind of `<` condition.

And maybe the extra byte is actually superfluous.